### PR TITLE
Fix err messages

### PR DESCRIPTION
**Pull request by din-mukhammed**
https://github.com/0chain/zboxcli/pull/506 - main PR
https://github.com/0chain/gosdk/pull/1172
https://github.com/0chain/blobber/pull/1214

### CODE REVIEW

**Comment by Kishan-Dhakan** [comment]
@din-mukhammed can you link the relevant PRs in the description? Would like to help with a full run as this looks like it has lot of breaking changes/error code changes

File: .github/workflows/nightly-current-sprint.yaml [hunk 1]
```diff
@@ -12,7 +12,6 @@ jobs:
   system-tests:
     name: "System Tests"
     runs-on: [ tests-suite ]
-    timeout-minutes: 120
```
Comment (stewartie4) [inline .github/workflows/nightly-current-sprint.yaml:15]: we need some sort of timeout, is 2 hrs not enough?

File: tests/cli_tests/zboxcli_create_allocation_test.go [hunk 2]
```diff
@@ -150,7 +150,7 @@ func TestCreateAllocation(testSetup *testing.T) {
 		// upload for creating wallet should fail
 		output, err = uploadFile(t, configPath, uploadParams, false)
 		require.NotNil(t, err, strings.Join(output, "\n"))
-		require.Contains(t, strings.Join(output, ""), "Operation needs to be performed by the owner or the payer of the allocation")
+		require.Contains(t, strings.Join(output, ""), "consensus_not_met")
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_create_allocation_test.go:153]: The previous error message was more user friendly

File: tests/cli_tests/zboxcli_create_dir_test.go [hunk 3]
```diff
@@ -107,8 +107,8 @@ func TestCreateDir(testSetup *testing.T) {
 		require.NotNil(t, err, "expected create dir failure command executed with output: ", strings.Join(output, "\n"))
 		require.Len(t, output, 1)
 		aggregatedOutput := strings.ToLower(strings.Join(output, " "))
-		require.Contains(t, aggregatedOutput, "directory creation failed")
-		require.Contains(t, aggregatedOutput, "consensus not met")
+		require.Contains(t, aggregatedOutput, "createdir failed")
+		require.Contains(t, aggregatedOutput, "consensus_not_met")
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_create_dir_test.go:111]: same here

File: tests/cli_tests/zboxcli_create_dir_test.go [hunk 4]
```diff
@@ -238,7 +238,7 @@ func TestCreateDir(testSetup *testing.T) {
 		output, err := createDirForWallet(t, configPath, wallet, true, allocID, true, "", false)
 		require.NotNil(t, err, "Expecting create dir failure %s", strings.Join(output, "\n"))
 		require.Len(t, output, 1)
-		require.Equal(t, "CreateDir failed:  invalid_name: Invalid name for dir", output[0])
+		require.Equal(t, "CreateDir failed:  invalid_path: Path is not absolute", output[0])
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_create_dir_test.go:241]: again, the old message made more sense to the end user

File: tests/cli_tests/zboxcli_file_copy_test.go [hunk 5]
```diff
@@ -291,7 +291,7 @@ func TestFileCopy(testSetup *testing.T) { // nolint:gocyclo // team preference i
 		}, false)
 		require.NotNil(t, err, strings.Join(output, "\n"))
 		require.Len(t, output, 1)
-		require.Contains(t, output[0], "copy_failed")
+		require.Contains(t, output[0], "Error performing CopyObject")
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_file_copy_test.go:294]: This is leaking internal info to the end user

**Review by stewartie4 — CHANGES_REQUESTED** [review]
some queries

File: .github/workflows/nightly-current-sprint.yaml [hunk 6]
```diff
@@ -12,6 +12,7 @@ jobs:
   system-tests:
     name: "System Tests"
     runs-on: [ tests-suite ]
+    timeout-minutes: 120
```
Comment (Kishan-Dhakan) [inline .github/workflows/nightly-current-sprint.yaml:15]: remove this

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**